docker/linux/arm64v8/Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM golang:1.15
 
 # download the source
 WORKDIR /go/src/github.com/nats-io/prometheus-nats-exporter
-RUN git clone --branch v0.6.0 https://github.com/nats-io/prometheus-nats-exporter.git .
+RUN git clone --branch v0.7.0 https://github.com/nats-io/prometheus-nats-exporter.git .
```

Comment:
btw would this address some of this? https://github.com/nats-io/k8s/issues/197

Reply:
I think no. As far as I can tell, we don't have automatic deployment of Docker images. We'll have to use `buildx` to build the arm64v8 image manually. However, I can look into this. 👍🏽